src/Paket.Core/TemplateFile.fs

```diff
@@ -6,6 +6,80 @@ open System.IO
 open System.Text.RegularExpressions
 open Paket.Rop
 open Paket.Domain
+
+module private TemplateParser =
+    type private ParserState =
+        {
+            Remaining : string list
+            Map : Map<string, string>
+            Line : int
+        }
+
+    let private single = Regex("^(\S+)\s*$", RegexOptions.Compiled)
```

Comment:
In C#, it should definitely be `Regex(@"^(\S+)\s*$")` because `\S` should be passed to the regex engine, therefore you'd need `"\\S"` — or `@"\S"`, of course.